lua/src/broker_cache.cc

```diff
@@ -98,6 +133,7 @@ void broker_cache::broker_cache_reg(lua_State* L, macro_cache const& cache) {
     { "__gc", l_broker_cache_destructor },
     { "get_hostname", l_broker_cache_get_hostname },
     { "get_service_description", l_broker_cache_get_service_description },
+    { "get_index_mapping", l_broker_cache_get_index_mapping },
```

Comment:
Could you also add _get_metric_mapping_ and _get_instance_ so that all methods of the macro_cache are available ?